ctapipe/tools/merge.py

```diff
@@ -49,7 +37,9 @@
     "/dl1/event/telescope/images",
     "/dl1/service/image_statistics",
     "/dl1/service/image_statistics.__table_column_meta__",
+    "/dl2/event/subarray/geometry",
```

Comment:
Should we also support `/dl2/event/telescope/geometry` in this? ctapipe doesn't yet write it, but I could imagine that lstchain or similar might (if they are using the DataWriter, it would get written). 

 Or we can leave that until it is implemented in ctapipe (once there is a proper machine learning module for mono geometry, energy, etc)

Reply:
lstchain does not yet use the ctapipe format, so the merge tool is  not applicable anyways.

I though about it, but I went for only merging what we already have. If in future we implement something that would use it, we should make sure the code to merge it gets also added to the merge tool

Reply:
E.g. I will take care of it when implementing telescope-wise energy / gammaness prediction in the ml module

Reply:
Ok sounds good